### ISSUE TITLE
Text of previous entry shows up in "What's Up"-field

### ISSUE THREAD

**Issue by c-rohrbach** [open-nomie/nomie6-oss#28]
I have encountered a bug when I use trackers consecutively. After starting the app, the first note saves correctly. When I then use a second tracker without saving instantly (i.e. a tally tracker with "save on tap" disabled or by pressing "insert" on an input menu), the text that appears in the text field is wrong. Instead of the value of tracker nr. 2, it will show the text of tracker nr. 1, which has already been saved. It seems the app wrongly remembers this text, which I can only clear by restarting the app or switching to another menu tab e.g. History or Dash.

I'm using version 6.1 on my android phone, installed as an app via Chrome.

**Comment by donalmacanri** [comment]
I have definitely encountered this before as well. But of course, now that I'm trying to reproduce it, it's not happening for me! Could you check is it still consistently happening for you on the latest build?

**Comment by c-rohrbach** [comment]
I switched over to the newest version and the issue is still showing up, but I think it didn't immediately.
I opened a new one (without any data) in a seperate browser, added some random trackers and input values and couldn't get it to show up. But it popped again after importing a data backup.
I tried again, new blank slate. Then installed the ADHD template and it was happening immediately. So maybe it is in some strange way connected to the import? Seems like a tough one for sure.
